tests/JustSaying.UnitTests/JustSaying.UnitTests.csproj

```diff
@@ -17,5 +17,9 @@
     <PackageReference Include="Polly" Version="7.2.1" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
+    <PackageReference Include="Quibble.Xunit" Version="0.2.51" />
```

Comment:
Doesn't build with this as it's not strong-named.

Reply:
I wept a little when I saw the error